app/src/main/java/com/ncnf/feed/ui/EventAdapter.java

```diff
@@ -59,7 +59,12 @@ public EventAdapter(List<Event> items, OnEventListener onEventListener, SortingM
     }
 
     public List<Event> getEvents() {
-        return events;
+        return Collections.unmodifiableList(events);
+    }
+
+    public void setEvents(List<Event> events) {
+        this.events = events;
```

Comment:
Savage unchecked override

Reply:
Sure, I will try to change the whole Adapter once this is merged.